docs/governance/community-guidelines/releases/README.md

```diff
@@ -0,0 +1,81 @@
+# Releases
+
+This document is a [community guideline].
+
+## Purpose
+
+The document outlines the guidelines of managing releases within the Common Cloud Controls (CCC) project.
+
+## Ownership
+
+The [Communications WG] has overall ownership of releases. They are responsible for setting the roadmap and communicating project progress to the wider community. All other [WG]'s should contribute to determining the content of a release.
+
+## Scheduling
+
+The community aims for several releases per year. Releases can occur as soon as the controls for a service are complete and approved. As the CCC project is producing a standard, releases should encapsulate a concrete set of stable, robustly tested changes constituting a new version of the project over small isolated changes and bugfixes.
+
+### Events
+
+The community can align releases with events such as the [Open Source in Finance Forum (OSFF)](https://events.linuxfoundation.org/open-source-finance-forum/). These events provide excellent opportunities to showcase the project and generate interest.
+
+## Project Board Management
+
+To manage GitHub issues targeted for a release, we recommend using a combination of [Milestones] and labels.
+
+### Milestones
+
+[Milestones] group GitHub issues related to a specific release. Each release should have a single top-level milestone associated with all relevant issues.
+
+There are two types of releases:
+
+1. Service Releases: Controls for individual services can be released independently.
+2. Service Sets: When appropriate, the controls of multiple services can be grouped together and released as a bundle.
+
+### Labels
+
+Each [WG] should apply their label to relevant issues, clearly indicating which issues fall under their responsibility.
+
+Additionally, labels can flag issues specifically targeted for events (e.g., OSFF events).
+
+## Release Process
+
+The release process involves contributors proposing changes through a pull request, followed by reviews and validations from relevant Working Groups, the creation of a release candidate, and multiple stages of approval from the [Change Management Board] (CMB) and project contributors, culminating in the official release being published and announced to the community. The diagram below explains the entire flow at a high-level, and we expand upon this in the subsections of the flow diagram.
+
+### Release Flow Diagram
+
+![Release Flow Diagram](./imgs/release-process.drawio.svg)
+
+### Flow Explained
+
+#### Pull Request Validation
+
+1. **Submission:** A contributor proposes a deliverable or change through a Pull Request (PR) to the CCC repository.
+2. **Working Group Review:** The relevant [WG]s will review the PR.
```

Comment:
Need to refine this section...

We haven't been requiring that every PR be ready for release. Perhaps we should make an Issue Template and have that be the trigger for final review, instead of a PR?